dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -1,13 +1,18 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="15.0">
 
-  <!-- With .NET 5, the propery TargetPlatformIdentifier is not UAP but rather windows.
-       Likewise, for other console apps, TargetPlatformIdentifier is windows.
-       This file expects UAP for all .NET versions, so adjust. -->
+  <!-- With .NET 5 and above, the propery TargetPlatformIdentifier is not UAP but rather Windows.
+       Likewise, for other console apps, TargetPlatformIdentifier is Windows.
+       PRI generation in VS apps was developed for UWPs originally. Some of the binaries involved assume
+       that the app is a UWP. The implementation needs to be changed but that'll likely happen as part of
+       moving said binaries to be out-of-box. For now, change the value provided to said binaries to UAP. -->
```

Comment:
I would avoid talking implementation roadmap in these targets. Just explaining the issue should be sufficient,